python/py-gwpy/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        gwpy gwpy 0.7.0 v
+name                py-gwpy
+
+categories-append   science
+maintainers         {ligo.org:duncan.macleod @duncanmmacleod}
```

Comment:
Can openmaintainer be added?

Reply:
@duncanmmacleod ?

Reply:
yes, what's the significance of `openmaintainer`? (@mf2k, @pmetzger)

Reply:
It means you are allowing other people to make changes when necessary without being consulted first.

Reply:
(You add it to the end of the maintainer line.)